integration-libs/cdc/manage-users/cdc-user-list.service.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { CdcJsService } from '@spartacus/cdc/root';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  WindowRef,
+} from '@spartacus/core';
+import { B2BUserService } from '@spartacus/organization/administration/core';
+import { TableService } from '@spartacus/storefront';
+import { UserListService } from '@spartacus/organization/administration/components';
+import { Subscription } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcUserListService extends UserListService {
+  constructor(
+    protected tableService: TableService,
+    protected userService: B2BUserService,
+    protected globalMessageService: GlobalMessageService,
+    protected winRef: WindowRef,
+    protected cdcJsService: CdcJsService
+  ) {
+    super(tableService, userService);
+  }
+  protected subscription: Subscription = new Subscription();
+
+  onCreateButtonClick(): void {
+    this.cdcJsService.didLoad().subscribe((cdcLoaded) => {
```

Comment:
`this.cdcJsService.didLoad()` check can be skipped, since the user would have logged in already using CDC SDK.